src/coreComponents/constitutive/contact/CoulombContact.cpp

```diff
@@ -47,6 +48,11 @@ CoulombContact::CoulombContact( string const & name, Group * const parent ):
     setApplyDefaultValue( -1 ).
     setInputFlag( InputFlags::OPTIONAL ).
```

Comment:
The trouble with letting users provide either the angle or the coefficient directly is that both are optional.  I might pick one and make it required.  The user can trivially do the conversion externally.